tests/integration.rs

```diff
@@ -0,0 +1,280 @@
+// Magical Bitcoin Library
+// Written in 2022 by
+//     Rajarshi Maitra <rajarshi149@gmail.com>
+//
+// Copyright (c) 2022 Magical Bitcoin
```

Comment:
I think we need to change the copyright and project name in this file (and others in this project) to:
```rust
// Copyright (c) 2020-2021 Bitcoin Dev Kit Developers
//
// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
// You may not use this file except in accordance with one or both of these
// licenses.
```
The others can be fixed in another PR but best to start this one out right.